workers/unity/Assets/Gdk/Core/Components/WorldCommandsTranslation.cs

```diff
@@ -8,29 +8,29 @@
 
 namespace Improbable.Gdk.Core
 {
-    internal struct CreateEntityRequest
+    public struct CreateEntityRequest
     {
         public Worker.Entity Entity;
         public Collections.Option<EntityId> EntityId;
```

Comment:
If this is public then we are generally trying to only expose Nullable rather than improbable collections. In our case I don't think that we actually have nullable in the public api either though.

Reply:
So what should be done here? What does codegen do with collections defined in schema (I assume we should mirror that here)?

Reply:
Codegen will either use a gdk option type or give you a nullable; I can't remember if we have switched the former yet. However in this api we can change things up a bit. Here it is acceptable to just use nullable or to use a long (probably nullable). The world translations will need removing or changing soon anyway, so I wouldn't worry too much about keeping this consistent with codegen.